src/main/kotlin/com/chattriggers/ctjs/minecraft/listeners/ClientListener.kt

```diff
@@ -180,6 +181,14 @@ object ClientListener : Initializer {
             }
         }
 
+        CTEvents.RENDER_BLOCK_ENTITY.register { stack, blockEntity, partialTicks, ci ->
+            renderTrigger(stack, partialTicks) {
+                // TODO(breaking): Don't pass the position into the trigger (they can get it
```

Comment:
Can we move this to `IRegister`? That's where most of the others are so might as well keep them all in the same spot.

Reply:
What others are in IRegister? Most of the CTEvents are inside ClientListener so that's why I put this there as well.

Reply:
I was referencing the other breaking comments in `IRegister`, but I guess there are others in this file anyways so it's not that big of a deal.

Reply:
Ah OK. Yeah I can move those over.